### PR TITLE
Automations. Support executionID for activateFlowTriggerById

### PR DESCRIPTION
**Pull request by viktor-liablin**
Automations. Support executionID for activateFlowTriggerById

### CODE REVIEW

File: src/automations/index.js [hunk 1]
```diff
@@ -71,9 +71,20 @@ export default class Automations {
       throw new Error('The "data" argument must be an object.')
     }
 
+    if (executionId !== undefined && typeof executionId !== 'string') {
+      throw new Error('The "executionId" argument must be a string.')
+    }
+
+    const query = {}
+
+    if (executionId !== undefined) {
```
Comment (stanislav-grin) [inline src/automations/index.js:80]: If an empty string is not a valid `executionId`, then we can (and should) check just for the `executionId` presence here:

```js
if (executionId) {
  query.executionId = executionId
}
```

File: test/unit/specs/automations/basic.js [hunk 2]
```diff
@@ -262,6 +274,18 @@ describe('<Automations> Basic', function() {
...
+    it('fails when execution id is invalid', async () => {
+      const errorMsg = 'The "executionId" argument must be a string.'
+
+      await expect(Backendless.Automations.activateFlowTriggerById(FLOW_ID, TRIGGER_ID, {}, null)).to.eventually.be.rejectedWith(errorMsg)
+      await expect(Backendless.Automations.activateFlowTriggerById(FLOW_ID, TRIGGER_ID, {}, true)).to.eventually.be.rejectedWith(errorMsg)
+      await expect(Backendless.Automations.activateFlowTriggerById(FLOW_ID, TRIGGER_ID, {}, false)).to.eventually.be.rejectedWith(errorMsg)
+      await expect(Backendless.Automations.activateFlowTriggerById(FLOW_ID, TRIGGER_ID, {}, 0)).to.eventually.be.rejectedWith(errorMsg)
+      await expect(Backendless.Automations.activateFlowTriggerById(FLOW_ID, TRIGGER_ID, {}, 123)).to.eventually.be.rejectedWith(errorMsg)
+      await expect(Backendless.Automations.activateFlowTriggerById(FLOW_ID, TRIGGER_ID, {}, [])).to.eventually.be.rejectedWith(errorMsg)
+      await expect(Backendless.Automations.activateFlowTriggerById(FLOW_ID, TRIGGER_ID, {}, () => ({}))).to.eventually.be.rejectedWith(errorMsg)
```
Comment (stanislav-grin) [inline test/unit/specs/automations/basic.js:287]: Empty string will not be rejects though it's not a valid execution ID

File: src/automations/index.js (hunk 1 above)
Comment (viktor-liablin) [reply src/automations/index.js]: \+

File: test/unit/specs/automations/basic.js (hunk 2 above)
Comment (viktor-liablin) [reply test/unit/specs/automations/basic.js:287]: \+

**Pull request merged**